src/main/resources/avro/sequenceAnnotationmethods.avdl

```diff
@@ -0,0 +1,126 @@
+@namespace("org.ga4gh.methods")
+
+protocol SequenceAnnotationMethods {
+
+  import idl "common.avdl";
+  import idl "methods.avdl";
+  import idl "sequenceAnnotations.avdl";
+
+  // FIXME: wiggle methods not defined.
+
+  /******************  /features/search  *********************/
+
+  /**
+  This request maps to the body of `POST /features/search` as JSON.
+  */
+  record SearchFeaturesRequest {
+    /**
+    The annotation set to search within. Either `featureSetId` or
+    `parentId` must be non-empty.
+    */
+    union { null, string } featureSetId;
+
+    /**
+    Restricts the search to direct children of the given parent `feature`
+    ID. Either `featureSetId` or `parentId` must be non-empty.
+    */
+    union { null, string } parentId;
+
+    /**
+    If specified, this query matches only annotations which overlap this path.
+    */
+    /* TODO: path not used yet, as graph code is not working in server.
+       Fields below are copied directly from variants.avdl for now.
+    union { null, org.ga4gh.models.Path } path = null; */
+
+    /**
+    Only return features with on the reference with this name.  One of this
+    field or `referenceId` is required.  (case-sensitive, exact match)
+    */
+    union { null, string } referenceName = null;
+
+    /**
+    Only return feature on the reference with this ID. One of this field or
+    `referenceName` is required.
+    */
+    union { null, string } referenceId = null;
```

Comment:
This needs to be consistent across the API for the moment.  Lets focus on being consistent for now.